### ISSUE TITLE
[CRITICAL] [FUCK] "Who" list not alphabetized

### ISSUE THREAD

**Issue by blobbernaut** [HippieStationCode/HippieStation13#2035]
fix

**Comment by Spacedong** [comment]
please fix this its useless as fuck without this

**Comment by Sarcalogo** [comment]
So you can see if your metabuddy is online??

**Comment by SplergeBucket** [comment]
yeah dude how am i meant to play the game if i cant see that meteoses, hackole, and kitty kathy are on so i can form a metagang

**Comment by Carbonhell** [comment]
should it be alphabetized or not then @nebyab

**Comment by Limed00d** [comment]
FIX

**Comment by Carbonhell** [comment]
so this is a bug?iunno it sounds normal but it can be easily fixed

**Comment by Sarcalogo** [comment]
Fix it so me and Kitty Kathy can be together for eternity unless some guy with a Kio-Kit name is online.

**Comment by nalarmg** [comment]
This is a bug, not a feature.
